examples/predictive_control_agent.py

```
@@ -0,0 +1,267 @@
+        self.best_strategy = None
+    
+    def try_strategy(self, strategy: dict) -> float:
+        """Try a strategy and return its output."""
+        state = State(
+            psi=strategy['psi'],
+            phi=strategy['phi'],
+            epsilon=strategy['epsilon'],
+            timestamp=strategy['time']
+        )
+        
+        result = self.engine.process_state(state)
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)